### PR TITLE
Changes for 2600 validation

### PR DESCRIPTION
**Pull request by yuvirani**
### Description
Changes for 2600 validation

### Check List
- [ ] New functionality includes testing.
  - [ ] All tests pass (`$ rake test`).
  - [ ] New resources and/or actions have are included in the [matchers.rb](https://github.com/HewlettPackard/oneview-chef/blob/master/libraries/matchers.rb) and [matchers_spec](https://github.com/HewlettPackard/oneview-chef/blob/master/spec/unit/resources/matchers_spec.rb).
- [ ] New functionality has been documented in the README if applicable.
  - [ ] New functionality has been thoroughly documented in [examples](https://github.com/HewlettPackard/oneview-chef/tree/master/examples/) (please include helpful comments).
- [ ] Changes documented in the [CHANGELOG](https://github.com/HewlettPackard/oneview-chef/blob/master/CHANGELOG.md).

### CODE REVIEW

File: endpoints-support.md [hunk 1]
```diff
@@ -17,268 +17,268 @@
...
+|<sub>/rest/fc-networks/{id}</sub>                                                        | GET      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |
+|<sub>/rest/fc-networks/{id}</sub>                                                        | PATCH    | :heavy_minus_sign:   | :white_check_mark:   | :white_check_mark:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :white_check_mark:   | :white_check_mark:   |
```
Comment (VenkateshRavula) [inline endpoints-support.md]: PATCH is no more valid. mark it as ```heavy_minus``` for API2400 and 2600

File: endpoints-support.md [hunk 2]
```diff
@@ -17,268 +17,268 @@
...
-|<sub>/rest/logical-interconnects/bulk-inconsistency-validation</sub>                     | POST      | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:  | :heavy_minus_sign:   | :heavy_minus_sign:   | :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |
+|<sub>/rest/logical-interconnects</sub>                                                   | GET       | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   | :white_check_mark:  |:white_check_mark:  |:white_check_mark:   |
+|<sub>/rest/logical-interconnects/locations/interconnects</sub>                           | POST      | :heavy_minus_sign:   | :heavy_minus_sign:   |  :heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   |:heavy_minus_sign:   |:white_check_mark:  |:white_check_mark:   |
```
Comment (VenkateshRavula) [inline endpoints-support.md]: change to heavy_minus for api2400 and 2600

File: endpoints-support.md [hunk 3]
```diff
@@ -17,268 +17,268 @@
...
+|<sub>/rest/logical-interconnects/locations/interconnects</sub>                           | DELETE    | :heavy_minus_sign:   | :heavy_minus_sign:   |  :heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   |:heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   |:heavy_minus_sign:   |:white_check_mark:  |:white_check_mark:   |
```
Comment (VenkateshRavula) [inline endpoints-support.md]: change to heavy_minus for api2400 and 2600

Comment (yuvirani) [reply endpoints-support.md]: Done

**Review by VenkateshRavula — APPROVED** [review]

**Review by nabhajit-ray — APPROVED** [review]
LGTM

**Pull request merged**